frontend/src/features/admin-form/create/workflow/components/EmptyWorkflow.tsx

```diff
@@ -24,11 +24,12 @@ export const EmptyWorkflow = (): JSX.Element => {
       pt={{ base: '0.5rem', md: '2.75rem' }}
     >
       <Text textStyle="h2" as="h2">
-        Create a workflow to collect responses from multiple respondents
+        Create a workflow to collect responses from multiple respondents in the
+        same form submission
       </Text>
       <Text textStyle="body-1" mt="1rem">
         Assign respondents to specific steps, and control which fields they can
-        see and fill.{' '}
+        fill. Learn how to create a workflow.{' '}
```

Comment:
Does it make sense for "Learn how to create a workflow" to be repeated twice? Should the first "Learn how to create a workflow" be removed?